projects/1.18/assets/stoneworks/stoneworks/lang/zh_cn.json

```
@@ -0,0 +1,584 @@
+  "block.stoneworks.basalt_shingles": "瓦制玄武岩",
+  "block.stoneworks.basalt_tile_slab": "竹制玄武岩台阶",
+  "block.stoneworks.basalt_tile_stairs": "竹制玄武岩楼梯",
+  "block.stoneworks.basalt_tile_wall": "竹制玄武岩墙",
+  "block.stoneworks.basalt_tiles": "竹制玄武岩",
+  "block.stoneworks.blackstone_paver_slab": "黑石地砖台阶",
+  "block.stoneworks.blackstone_paver_stairs": "黑石地砖楼梯",
+  "block.stoneworks.blackstone_paver_wall": "黑石地砖墙",
+  "block.stoneworks.blackstone_pavers": "黑石地砖",
+  "block.stoneworks.blackstone_pillar": "黑石柱",
+  "block.stoneworks.blackstone_plate_slab": "切制黑石台阶",
+  "block.stoneworks.blackstone_plate_stairs": "切制黑石楼梯",
+  "block.stoneworks.blackstone_plate_wall": "切制黑石墙",
+  "block.stoneworks.blackstone_plates": "切制黑石",
+  "block.stoneworks.blackstone_shingle_slab": "瓦制黑石台阶",
+  "block.stoneworks.blackstone_shingle_stairs": "瓦制黑石楼梯",
+  "block.stoneworks.blackstone_shingle_wall": "瓦制黑石墙",
+  "block.stoneworks.blackstone_shingles": "瓦制黑石",
+  "block.stoneworks.blackstone_tile_slab": "竹制黑石台阶",
+  "block.stoneworks.blackstone_tile_stairs": "竹制黑石楼梯",
+  "block.stoneworks.blackstone_tile_wall": "竹制黑石墙",
+  "block.stoneworks.blackstone_tiles": "竹制黑石",
+  "block.stoneworks.calcite_brick_slab": "方解石砖台阶",
+  "block.stoneworks.calcite_brick_stairs": "方解石砖楼梯",
+  "block.stoneworks.calcite_brick_wall": "方解石砖墙",
+  "block.stoneworks.calcite_bricks": "方解石砖",
+  "block.stoneworks.calcite_paver_slab": "方解石地砖台阶",
+  "block.stoneworks.calcite_paver_stairs": "方解石楼梯",
+  "block.stoneworks.calcite_paver_wall": "方解石地砖墙",
+  "block.stoneworks.calcite_pavers": "方解石地砖",
+  "block.stoneworks.calcite_pillar": "方解石柱",
+  "block.stoneworks.calcite_plate_slab": "切制方解石台阶",
+  "block.stoneworks.calcite_plate_stairs": "切制方解石楼梯",
+  "block.stoneworks.calcite_plate_wall": "切制方解石墙",
+  "block.stoneworks.calcite_plates": "切制方解石",
+  "block.stoneworks.calcite_shingle_slab": "瓦制方解石台阶",
+  "block.stoneworks.calcite_shingle_stairs": "瓦制方解石楼梯",
+  "block.stoneworks.calcite_shingle_wall": "瓦制方解石墙",
+  "block.stoneworks.calcite_shingles": "瓦制方解石",
+  "block.stoneworks.calcite_tile_slab": "竹制方解石台阶",
+  "block.stoneworks.calcite_tile_stairs": "竹制方解石楼梯",
+  "block.stoneworks.calcite_tile_wall": "竹制方解石墙",
+  "block.stoneworks.calcite_tiles": "竹制方解石",
+  "block.stoneworks.chiseled_andesite": "錾制安山岩",
+  "block.stoneworks.chiseled_basalt": "錾制玄武岩",
+  "block.stoneworks.chiseled_calcite": "錾制方解石",
+  "block.stoneworks.chiseled_dark_prismarine": "錾制暗海晶石",
+  "block.stoneworks.chiseled_diorite": "錾制闪长岩",
+  "block.stoneworks.chiseled_end_stone": "錾制末地石",
+  "block.stoneworks.chiseled_granite": "錾制花岗岩",
+  "block.stoneworks.chiseled_prismarine": "錾制海晶石",
+  "block.stoneworks.chiseled_purpur": "錾制紫珀块",
+  "block.stoneworks.chiseled_tuff": "錾制凝灰岩",
+  "block.stoneworks.cobbled_andesite": "碎状安山岩",
+  "block.stoneworks.cobbled_andesite_slab": "碎状安山岩台阶",
+  "block.stoneworks.cobbled_andesite_stairs": "碎状安山岩楼梯",
+  "block.stoneworks.cobbled_andesite_wall": "碎状安山岩墙",
```

```suggestion
  "block.stoneworks.cobbled_andesite": "安山岩圆石",
  "block.stoneworks.cobbled_andesite_slab": "安山岩圆石台阶",
  "block.stoneworks.cobbled_andesite_stairs": "安山岩圆石楼梯",
  "block.stoneworks.cobbled_andesite_wall": "安山岩圆石墙",
```
见原版`深板岩圆石`/`Cobbled Deepslate`
全文修改